lineapy/instrumentation/tracer.py

```diff
@@ -468,6 +468,7 @@ def assign(
         """
         logger.debug("assigning %s = %s", variable_name, value_node)
         self.variable_name_to_node[variable_name] = value_node
+        self.db.write_assigned_variable(value_node.id, variable_name)
```

Comment:
When there is mutation, would you just have a one to many relationship? I think it could be fine; just wanted to be very clear.

Reply:
I don't think this implementation is tracking mutation of a node. 

It does go through `assign`
https://github.com/LineaLabs/lineapy/blob/caf8604674eaf2f4178f239fc7f108d9ee54d880/lineapy/transformer/node_transformer.py#L399-L402

Reply:
I think I was overthinking it in the previous review. I think I agree with you that we don't need to worry too much about mutation here.

Good catch on the warus operator! Could you add a test case for `a,b = [1,2]`

Reply:
or (1,2)

Reply:
Added new tests